src/Twig/RichEditorExtension.php

```diff
@@ -26,48 +26,56 @@
 
 final class RichEditorExtension extends AbstractExtension
 {
-    /**
-     * @var RegistryInterface
-     */
-    private $uiElementRegistry;
+    private RegistryInterface $uiElementRegistry;
 
-    /**
-     * @var Environment
-     */
-    private $twig;
+    private Environment $twig;
```

Comment:
Why did you remove PHP Doc ?

Reply:
I removed it because the PHP 7.4 strict typing is enough, isn't it ?

Reply:
My bad it's good